### DISCUSSION TITLE
can anyone help to compile the ProRender blender add-on( v3.5.2) source code to to windows.zip ?

### DISCUSSION THREAD

**Discussion by chrisx2ds** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#659] [Q&A] [answered]
the source code  is here   https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/releases/tag/v3.5.2

can anyone help to compile it for windows?

Thanks

**Comment by bsavery** [comment]
Hey is there a reason why that release in particular is needed?  There are compilation instructions https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon

**Reply by chrisx2ds** [reply]
I am not a tech guy who know coding, thats why I ask for help who can compile the file for windows version of these blender plug-in.
can you help compile the file for me?

**Reply by bsavery** [reply]
Is there a reason you need this particular release?  There are newer ones that are ready compiled.

**Reply by chrisx2ds** [reply]
these is the latest version that support OpenCL , (in short, support on old amd card, mine is rx580)   :  https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/releases/tag/v3.5.2 

can you help compile?

**Reply by bsavery** [reply, marked as answer]
The current release also supports opencl under legacy mode:
1.  Set the Viewport AND final render settings use "RPR Final"
2.  Under the debug settings set it to use OpenCL

You can do this just as well with the current version.

**Reply by chrisx2ds** [reply]
oh, thanks
